### PR TITLE
Fix the '\n' problem

### PR DESCRIPTION
**Pull request by yashk2810**

### CODE REVIEW

File: guides/md/training_keras_models_on_cloud.md [hunk 1]
```diff
@@ -229,55 +258,65 @@ to monitor performance.
...
 TensorFlow Cloud is also runnable from Python notebooks. Additionally, your specified
 `entry_point` can be a notebook if needed. There are two key differences to keep
 in mind between TensorFlow Cloud on notebooks compared to scripts:
 
 - When calling `run()` from within a notebook, a Cloud Storage bucket must be specified
 for building and storing your Docker image.
 - GCloud authentication happens entirely through your authentication key, without
-project specification. 
-
-An example workflow using TensorFlow Cloud from a notebook
```
Comment (fchollet) [inline guides/md/training_keras_models_on_cloud.md:268]: That last sentence should be separate from the list, I believe

Comment (yashk2810) [reply guides/md/training_keras_models_on_cloud.md:268]: That's not how its in the SOT: https://github.com/keras-team/keras-io/blob/master/guides/training_keras_models_on_cloud.py#L273

Comment (yashk2810) [reply guides/md/training_keras_models_on_cloud.md:268]: If this needs to be a separate point, I can send a separate PR but I need to get this in to fix the tf.org view.

Comment (fchollet) [reply guides/md/training_keras_models_on_cloud.md:268]: I'm pretty sure it needs to be a separate point, let's fix it in a separate PR to the SoT

**Pull request merged**